### PR TITLE
Fix #2490 : Reformatted administrator controls app information view files

### LINKED ISSUE #2490: Required Reformatting for administrator_controls_app_information_view.xml
The following files require reformatting
 - layout/administrator_controls_app_information_view.xml
 - layout-land/administrator_controls_app_information_view.xml
 - layout-sw600dp/administrator_controls_app_information_view.xml 

Recommended to set up [Macros](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros) to solve this issue

References
 - https://github.com/oppia/oppia-android/wiki/Coding-style-guide#code-formatting
 - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

### PR DESCRIPTION
**Pull request by Mehul-Bisht**
## Explanation

Fixes #2490 Required Reformatting for administrator_controls_app_information_view.xml

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -5,6 +5,22 @@
...
+      <option name="PACKAGES_IMPORT_LAYOUT">
+        <value>
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
       <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
```
Comment (FareesHussain) [inline .idea/codeStyles/Project.xml]: Revert the changes in this file

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@Mehul-Bisht please revert the changes generated by Android Studio
as mentioned in the 4th line in the check list

**Comment by rt4914** [comment]
@Mehul-Bisht Please update title so that it contains the issue number.
Check point 4 in wiki: https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

**Comment by Mehul-Bisht** [comment]
@rt4914 @FareesHussain I have made the changes please review.

**Review by FareesHussain — APPROVED** [review]
LGTM

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.
